<launch>

    <include file="$(find vrx_navigation)/launch/docking.launch"/>
     <!-- <group ns="wamv">


    <node pkg="vrx_tasks" type="missionControl.py" name="missionControl" output="screen">
     <node pkg="vrx_vision" name="placard_classifier" type="classify_placard_server.py" output="screen"/>
    </group> -->
</launch>
